### DISCUSSION TITLE
Need help following the USB Creation instructions. - Solved it was the VPN.

### DISCUSSION THREAD

**Discussion by Kareemsi** [sebanc/brunch#1669] [General]
I tried following the steps but I can't seem to get anything to update in my Windows Powershell.

Err:1 http://archive.ubuntu.com/ubuntu focal InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:2 http://security.ubuntu.com/ubuntu focal-security InRelease
  Temporary failure resolving 'security.ubuntu.com'
Hit:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease
Reading package lists... Done
Building dependency tree
Reading state information... Done
280 packages can be upgraded. Run 'apt list --upgradable' to see them.
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package cgpt

How do I get the updates and how do I install the cgpt package to make the image. Please help.

I was having issues getting my DNS to ping Google.com. But after I manually changed the ip address and DNS addy in my network settings in windows and restarted my computer I can now ping google.com. But I can't seem to get anything to download from Ubuntu.com

-------------Follow up it seems I can ping archive.ubuntu.com-----------------------------
PING archive.ubuntu.com (91.189.91.39) 56(84) bytes of data.
64 bytes from kazooie.canonical.com (91.189.91.39): icmp_seq=1 ttl=55 time=21.1 ms
64 bytes from kazooie.canonical.com (91.189.91.39): icmp_seq=2 ttl=55 time=15.1 ms
64 bytes from kazooie.canonical.com (91.189.91.39): icmp_seq=3 ttl=55 time=14.9 ms
64 bytes from kazooie.canonical.com (91.189.91.39): icmp_seq=4 ttl=55 time=19.9 ms
64 bytes from kazooie.canonical.com (91.189.91.39): icmp_seq=5 ttl=55 time=15.0 ms
^C
--- archive.ubuntu.com ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4003ms
rtt min/avg/max/mdev = 14.926/17.206/21.116/2.714 ms

But still can't get any updates from it.

**Comment by ameyjoshi3068** [comment]
What I can understand is you are using wsl2 right?

**Reply by Kareemsi** [reply]
Yup yup! 

I'm following the tutorial to make a USB on my Windows computer. 
https://github.com/sebanc/brunch/blob/main/install-with-windows.md
I'm at number 4 under **Prepare the Terminal**

I can't get anything do update. At first I was having issues connecting to the internet through the WSL2. But I manually added my ip settings on my windows network for the WSL connection and I was able to ping stuff on the internet. But for some reason it still didnt' want to do any update packages or install anything from Ubuntu online. 

I can Ping Ubuntu now but can't get anything to download from the updates.

**Reply by ameyjoshi3068** [reply]
This is an issue which is present in wsl2 itself
You may try https://github.com/microsoft/WSL/issues/5061
or simply google your issue

**Comment by Kareemsi** [comment]
Solved the problem by manually entering a ip address and a DNS address in vEthernet(WSL) in windows network settings, and turning off my VPN.

It seem the VPN was the last issue that was the problem.
